your_changes.patch

```diff
@@ -0,0 +1,81 @@
+ 			<var name="latentHeatFlux"/>
+@@ -193,5 +195,7 @@
+ 			<var name="velocityMeridionalSquared"/>
+ 			<var name="velocityZonalTimesTemperature"/>
+ 			<var name="velocityMeridionalTimesTemperature"/>
++	                <var name="velocityZonalTimesTemperature_GM"/>
++                        <var name="velocityMeridionalTimesTemperature_GM"/>	
+ 		</stream>
+ 	</streams>
+diff --git a/src/core_ocean/analysis_members/mpas_ocn_eddy_product_variables.F b/src/core_ocean/analysis_members/mpas_ocn_eddy_product_variables.F
+index d0b1ea9..b708108 100644
+--- a/src/core_ocean/analysis_members/mpas_ocn_eddy_product_variables.F
++++ b/src/core_ocean/analysis_members/mpas_ocn_eddy_product_variables.F
+@@ -184,7 +184,8 @@ contains
+ 
+       real (kind=RKIND), dimension(:), pointer :: ssh, SSHSquared
+       real (kind=RKIND), dimension(:,:), pointer :: velocityZonal, velocityMeridional, &
+-           velocityZonalSquared, velocityMeridionalSquared, velocityZonalTimesTemperature, velocityMeridionalTimesTemperature
++           velocityZonalSquared, velocityMeridionalSquared, velocityZonalTimesTemperature, velocityMeridionalTimesTemperature, &
++           velocityZonalTimesTemperature_GM, velocityMeridionalTimesTemperature_GM, GMBolusVelocityZonal, GMBolusVelocityMeridional
+       real (kind=RKIND), dimension(:,:,:), pointer :: activeTracers
+ 
+       err = 0
+@@ -216,6 +217,11 @@ contains
+          call mpas_pool_get_array(eddyProductVariablesAMPool, 'velocityZonalTimesTemperature', velocityZonalTimesTemperature)
+          call mpas_pool_get_array(eddyProductVariablesAMPool, 'velocityMeridionalTimesTemperature', &
+             velocityMeridionalTimesTemperature)
++         call mpas_pool_get_array(eddyProductVariablesAMPool,'velocityZonalTimesTemperature_GM', velocityZonalTimesTemperature_GM)
++         call mpas_pool_get_array(eddyProductVariablesAMPool,'velocityMeridionalTimesTemperature_GM', &
++            velocityMeridionalTimesTemperature_GM)
++         call mpas_pool_get_array(diagnosticsPool, 'GMBolusVelocityZonal', GMBolusVelocityZonal)
++         call mpas_pool_get_array(diagnosticsPool, 'GMBolusVelocityMeridional', GMBolusVelocityMeridional)
+ 
+          do iCell = 1,nCellsSolve
+             SSHSquared(iCell) = ssh(iCell)**2
+@@ -224,6 +230,8 @@ contains
+                velocityMeridionalSquared(k,iCell) = velocityMeridional(k,iCell)**2
+                velocityZonalTimesTemperature(k,iCell) = velocityZonal(k,iCell)*activeTracers(index_temperature,k,iCell)
+                velocityMeridionalTimesTemperature(k,iCell) = velocityMeridional(k,iCell)*activeTracers(index_temperature,k,iCell)
++               velocityZonalTimesTemperature_GM(k,iCell) = GMBolusVelocityZonal(k,iCell)*activeTracers(index_temperature,k,iCell)
++               velocityMeridionalTimesTemperature_GM(k,iCell) = GMBolusVelocityMeridional(k,iCell)*activeTracers(index_temperature,k,iCell)
+             end do
+          end do
+ 
```

Comment:
@lconlon, please remove this from the commit history.

Reply:
Should be gone now. Thanks for catching that!